cmd/run/run.go

```diff
@@ -179,6 +179,12 @@ func NewRunCommand() *cobra.Command {
 
 	flags.Uint32("listObjects-max-results", defaultConfig.ListObjectsMaxResults, "the maximum results to return in non-streaming ListObjects API responses. If 0, all results can be returned")
 
+	flags.Bool("check-query-cache-enabled", defaultConfig.CheckQueryCache.Enabled, "when executing Check and ListObjects requests, enables caching. This will turn Check and ListObjects responses into eventually consistent responses")
```

Comment:
Can we please add tests for these new config options like we've done here?

https://github.com/openfga/openfga/blob/main/cmd/run/run_test.go#L1079-L1120

We should test to make sure that the config file, env variables, and flags all behave as expected by asserting that `ReadConfig` parses the provided configs correctly (including the merging of values behavior).